Lib/platform.py

```diff
@@ -1344,6 +1349,121 @@ def freedesktop_os_release():
     return _os_release_cache.copy()
 
 
+### musl libc version support
+# These functions were copied and adapted from the packaging module:
+# https://github.com/pypa/packaging/blob/main/src/packaging/_musllinux.py
+# https://github.com/pypa/packaging/blob/main/src/packaging/_elffile.py
```

Comment:
nit: If the packaging module switches to use this new stdlib code at some point, the references might become invalid. A possible solution would be to use permalinks.